src/test/controller-tests/user.http

```diff
@@ -0,0 +1,16 @@
+POST http://localhost:8080/v1/user
+Content-Type: application/json
+
+{}
```

Comment:
tutaj cos powinnismy podac, chcemy utworzyc usera

Reply:
Racja, przeoczyłem to